contracts/mixnet/src/mixnodes/transactions.rs

```diff
@@ -152,6 +154,8 @@ fn _try_add_mixnode(
         storage::TOTAL_DELEGATION.save(deps.storage, identity, &Uint128::zero())?;
     }
 
+    storage::LAST_PM_UPDATE_TIME.save(deps.storage, identity, &env.block.time.seconds())?;
```

Comment:
We should probably also save into this map when bonding mixnode, so somebody wouldn't be able to bond with low profit margin and then, before 30 days are over update it to a way higher one.